adapter/PreciceInterface.h

```diff
@@ -55,7 +55,7 @@ typedef struct PreciceInterface {
 
 	// Arrays to store the coupling data
 	double * nodeScalarData;
-        double * nodeVectorData; //Forces, displacements and displacementDeltas are vector quantities
+    double * nodeVectorData; //Forces, displacements and displacementDeltas are vector quantities
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).